source/code-snippets/crud/cursor.js

```diff
@@ -68,6 +68,17 @@ async function close(myColl) {
   await cursor.close();
   // end close cursor example
 }
+// Abort in-progress operations
+async function abort(myColl) {
+  // start abort cursor example
+  const abort = new AbortController();
+  const { signal } = abort;
```

Comment:
Just took a look at the JIRA ticket and wondering if this is the proper syntax?
```suggestion
  const signal = abort.signal;
```

Reply:
It's the same result either way but code samples almost always use the [destructuring syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) when demonstrating abort signal use